Frontend/src/app/app-routing.module.ts

```diff
@@ -9,8 +9,13 @@ import { ProfileComponent } from './views/dashboard/profile/profile.component';
 import { TimeComponent } from './views/dashboard/time/time.component';
 import { AuthGuard } from './core/guards/auth.guard';
 import { ProjectDetailsComponent } from './views/dashboard/project-details/project-details.component';
+import { HomeComponent } from './shared/components/home/home.component';
 
 const routes: Routes = [
+  {
+    path: 'home',
+    component: HomeComponent
+  },
```

Comment:
We should display the HomeComponent as the main component on the website. Should be `website.com` instead of `website.com/home`.

Reply:
Corrected. Caused navbar switch issue, to be fixed in navbar PR.